site-src/concepts/guidelines.md

```diff
@@ -104,11 +104,26 @@ mostly but not entirely portable.
 
 ## API Conventions
 
-Service APIs follow Kubernetes API [conventions][1]. These conventions
+Gateway API follows Kubernetes API [conventions][1]. These conventions
 are intended to ease client development and ensure that configuration
 mechanisms can consistently be implemented across a diverse set of use
-cases. One variation from the Kubernetes conventions is for fields with
-default values. These fields are considered optional but are not pointers
+cases. In addition to the Kubernetes API conventions, Gateway API has the
+following conventions:
+
+### Fields with default values
+
+These fields are considered optional but are not pointers
 since they will never be empty.
```

Comment:
Sorry for the confusion here, I think this particular point is no longer the case. We went back and forth on this before eventually settling on pointers for optional values, even if they did have a default value. For example: https://github.com/kubernetes-sigs/gateway-api/blob/master/apis/v1alpha2/gateway_types.go#L324-L326

Reply:
+1

Reply:
Okay, I'll just remove this whole bit then, no worries.